src/pages/Home/Home.js

```diff
@@ -67,7 +67,7 @@ const Home = ({ data }) => {
         </Container>
       </section>
 
-      <section className="home__section">
+      <section className="home__section" id="newsletters">
```

Comment:
not sure why you are adding this

Reply:
its for linking to this section but its not being used right will still keep since in the future this will be here